src/mobile/src/components/Banner/Banner.jsx

```diff
@@ -0,0 +1,36 @@
+import { View, StyleSheet } from 'react-native';
+import BannerButtons from './BannerButtons';
+import BannerText from './BannerText';
+
+const styles = StyleSheet.create({
+  container: {
+    backgroundColor: '#121D59',
+    flex: 1,
+    opacity: 0.81,
+    padding: 22,
+  },
+  backdrop: {
+    overflow: 'hidden',
+    position: 'absolute',
+    top: 0,
+    left: 0,
+    bottom: 0,
+    right: 0,
+    boxSizing: 'border-box',
+    margin: 0,
+    backgroundColor: '#000000',
+    opacity: 0.8,
+  },
+});
+
+const Banner = ({ navigateToContact }) => {
+  return (
+    <View style={styles.container}>
```

Comment:
We could do a follow-up issue to fetch images from our telescope API.

Reply:
> We could do a follow-up issue to fetch images from our telescope API.
True. right now the banner does not have image service